apertium_apy/apy.py

```diff
@@ -178,11 +185,17 @@ def apply_config(args, parser, apy_section):
 
 
 def parse_args(cli_args=sys.argv[1:]):
+    if(found is False):
+        location = langNamesDefault
+    else:
+        path = pkg_resources.resource_filename('apertium_apy', langNamesDefault)
+        location = os.path.join(os.path.dirname(os.path.dirname(path)), langNamesDefault)
```

Comment:
These variable names should be more specific. Also, what does this stuff with `os.path.dirname` actually do?

Reply:
Okay I will use more appropriate variable names. `os.path.dirname` gives the directory of specified path. Using it 2 times take me 2 directories back and then I am joining the `langNames` to give correct path.

Reply:
Why isn't the line above from `pkg_resources` giving you the desired path?

Reply:
pkg_resources work on installed module i.e. apertium_apy. It give me path including apertium_apy. So i have to come out of that directory as langNames should be present there. This is how pkg_resouces work :)

Reply:
In that case, what does giving it `langNamesDefault` achieve?

Reply:
I have updated langNamesDefault to langNamesDeafultPath. It contains the name of langNames.db. If pkg_resources is installed then default location is given to langNames.db and else default location is recognised by pkg_resources. Please review the updated files this one's outdated.